### DISCUSSION TITLE
Multiple instance

### DISCUSSION THREAD

**Discussion by mmeyer56** [jendrikseipp/rednotebook#736] [Q&A] [answered]
It has become impossible to open multiple instances since the last version on Windows. Is it a wish or a bug ? Or am I only one ?

**Comment by jendrikseipp** [comment, marked as answer]
This is by design. Having multiple instances open would be a recipe for data loss currently. It would be a different story if different instances only allowed to have different journals open, but this cannot be enforced at this moment.
